src/tokens.js

```diff
@@ -803,13 +797,26 @@ export function trackVariables(context = {}) {
         const children = variables.children.slice(0, -1);
         const lastChild = variables.children.slice(-1)[0];
 
+        let newContext = { ...variables.context, ...lastChild?.computedValue() };
+
+        if (term === pathExpressionStart) {
+          newContext = {
```

Comment:
Needs test coverage, cf. https://github.com/nikku/lezer-feel/pull/16#issuecomment-1400362531.

Reply:
tested in [858b70d](https://github.com/nikku/lezer-feel/pull/16/commits/858b70dfe81cb7035333bea866f12d2458205d23)

Reply:
:star: